src/App.css

```diff
@@ -30,3 +30,12 @@
     transform: none;
   }
 }
+.card-container {
+  height: 325px;
+}
+.title-container {
+  height: 100px;
```

Comment:
tw has some classes for height `h-24` , `h-80` 

https://tailwindcss.com/docs/height

useful tailwind vscode extension: https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss